azavea/district-builder-dtl-pa
# Enable base map switcher

## Overview

This enables the base map switcher and adds a couple new types of base maps.

### Checklist


### Demo

![pa-basemap-switcher](https://user-images.githubusercontent.com/6386/38514281-c7ff0fc8-3bfe-11e8-9ee9-c706b33b1562.png)

### Notes

Probably best to look at the commits separately, since the `Formatting` commit was just a trailing whitespace getting automatically trimmed.

## Testing Instructions

 * I believe you can get this working by just SSHing into the VM and running `scripts/update`. This should copy the basemap settings over to the appropriate place.
 * Then just load the map and verify that each of the basemap options works
